lib/actions/user-actions.js

```diff
@@ -290,6 +290,18 @@ async function requestAccess(
   await fetchJSON('request_access', accessRequest);
 }
 
+const sendFriendRequestActionTypes = Object.freeze({
+  started: 'SEND_FRIEND_REQUEST_STARTED',
+  success: 'SEND_FRIEND_REQUEST_SUCCESS',
+  failed: 'SEND_FRIEND_REQUEST_FAILED',
+});
+async function sendFriendRequest(
```

Comment:
Do you think it would be better to have the server endpoint be more generic? If the backend treats `FRIEND` as a generic case of a relationship, and returns `UserInfo` objects with a field that says `FRIEND`, maybe we could have a single endpoint for handling both friend requests and blocks

Reply:
I agree. I'm going to create a generic endpoint. That was a temporary solution, just wanted to play a bit with client-server communication. I think this endpoint should not be included in this PR since it's all about UI. I can remove it for now

Reply:
Is the plan to remove the endpoint then?